recipes/osx-sysroot/conda_build_config.yaml

```diff
@@ -0,0 +1,51 @@
+_MACOSX_DEPLOYMENT_TARGET_:
+  - "10.9"
+  - "10.13"
+  - "11.0"
+  - "11.0"
+  - "12.3"
+
+cross_target_platform:
+  - osx-64
+  - osx-64
+  - osx-64
+  - osx-arm64
+  - osx-arm64
+
+_MACOSX_SDK_VERSION_:
+  - "1"
+  - "10.9"
+  - "10.13"
+  - "11.0"
+  - "12.3"
+
+is_default_sdk:
+  - "no"
+  - "no"
+  - "yes"
+  - "yes"
+  - "no"
+
+is_arm64:
+  - "yes"
+  - "no"
+  - "no"
+  - "yes"
+  - "yes"
```

Comment:
I find it weird to have a zip with `cross_target_platform` and then another one with `is_arm64`. I get that we need to zip the deployment target and sdk separately, but the CBC is really hard to digest IMO.

Given the constraint of "no selectors" for keeping it noarch, perhaps its worth considering to build the sdk and the deployment target on separate feedstocks? That way each of them could have a cleaner CBC, and we could still express dependencies / run_constraints between them.